### PR TITLE
feat: add features section

### PR DESCRIPTION
**Pull request by thestuti**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,6 +50,50 @@ <h1>Mantra Bot</h1>
...
+        </div>
+        <div class="row">
+          <div class="column">
+            <div class="card">
+              <div class="icon">
+                <i class="fa-solid fa-user"></i>
+              </div>
+              <h3>User Friendly</h3>
+              <p>
+                Mantra Bot offers a lot of features and commands, and it is very user-friendly. To make things simple for users, we implemented the slash command in this bot.
```
Comment (thenishantsapkota) [inline index.html:66]: `slash commands`

File: index.html [hunk 2]
```diff
@@ -48,6 +50,50 @@ <h1>Mantra Bot</h1>
...
+              </p>
```
Comment (thenishantsapkota) [inline index.html:67]: Also link to the mantra bot readme for now until we build the command page .

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Comment by thenishantsapkota** [comment]
Please make the requested changes.
Other than that the UI looks super cool 😎
Thank you

**Review by thenishantsapkota — APPROVED** [review]

**Comment by thenishantsapkota** [comment]
Looks fabulous to me🥳

**Comment by thestuti** [comment]
Thank you 😁

**Pull request merged**